resources/js/components/Home/AuthBanner.vue

```diff
@@ -0,0 +1,70 @@
+<script setup lang="ts">
+import { ref, onMounted, onUnmounted, type Ref } from 'vue'
+import { inject } from 'vue'
+const isLoginRef: Ref<any, any> | undefined = inject('isLoginRef')
+
+
+// const props = defineProps(['isLogin'])
+
+// function gotoAuthSection() {
+// props.authForm.authForm
+// authFormRef.value.authForm
+
+// props.authForm.authForm.value.authForm.scrollIntoView({
+//   behavior: 'smooth',
+//   block: 'center', 
+// })
+
+// }
+
+const isHidden = ref(false)
+
+function setBannerVisibility() {
+  isHidden.value = window.scrollY / 100 >= 5
+}
+onMounted(() => window.addEventListener('scroll', setBannerVisibility))
+onUnmounted(() => window.removeEventListener('scroll', setBannerVisibility))
+
+function gotoAuthSection() {
+  window.scrollTo({
+    top: window.innerHeight,
+    behavior: 'smooth',
+  })
+}
+
+function gotoRegisterSection() {
+  gotoAuthSection()
+  if (isLoginRef)
+    isLoginRef.value.isLogin = false
+}
+
+function gotoLoginSection() {
+  gotoAuthSection()
+  if (isLoginRef)
+    isLoginRef.value.isLogin = true
+}
+
+</script>
+
+<template>
+  <div>
+    <div class="h-16 lg:hidden" />
+    <div :class="{ '-translate-y-full': isHidden }"
+         class="lg:hidden top-0 duration-200 justify-evenly z-10 backdrop-blur-md bg-white/30 lg:bg-white fixed w-full flex items-center gap-x-6 px-6 py-2.5"
+    >
+      <div class="justify-center flex-1 hidden lg:flex" />
+      <div class="hidden lg:justify-center flex-1 xs:flex">
+        <p class="text-sm leading-6 text-gray-900">Collegium Witelona Test </p>
```

Comment:
```suggestion
        <p class="text-sm leading-6 text-gray-900">Collegium Witelona Uczelnia Państwowa </p>
```

Reply:
Robimy stronę do testów, myślałem że może nazwać stronę "Collegium Witelona Testy"?

Reply:
I thought that you wrote "Test" just to test something, not to name the page with tests. So for now we can leave "Testy" - when we will implement frontend for tests, maybe we will rename it? We will see

Reply:
```suggestion
        <p class="text-sm leading-6 text-gray-900">Testy</p>
```